app/controllers/v1/interventions/stars_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class V1::Interventions::StarsController < V1Controller
+  def create
+    authorize! :read, Intervention
+
+    return head :not_found if intervention_load.nil?
+
+    current_v1_user.stars.find_or_create_by(intervention_id: intervention_load.id)
+
+    clear_cache
+
+    head :no_content
+  end
+
+  def destroy
+    authorize! :read, Intervention
+
+    return head :not_found if intervention_load.nil?
+
+    current_v1_user.stars.delete_by(intervention_id: intervention_load.id)
+
+    clear_cache
+
+    head :no_content
+  end
+
+  private
+
+  def interventions_scope
+    @interventions_scope ||= Intervention.accessible_by(current_ability)
+                                         .order(created_at: :desc).only_visible
+  end
+
+  def intervention_load
+    @intervention_load ||= interventions_scope.find_by(id: params[:intervention_id])
```

Comment:
Why `find_by` instead of `find`? `Find` will raise an exception which `app/controllers/concerns/exception_handler.rb` will handle